php/WP_CLI/CommandWithUpgrade.php

```diff
@@ -10,6 +10,13 @@
 	protected $upgrade_refresh;
 	protected $upgrade_transient;
 
+	function __construct() {
+		// Do not automatically check translations updates after updating plugins/themes.
```

Comment:
The `Language_Pack_Upgrader::async_upgrade` method returns HTML tags to the standard output.
I corrected this by adding a call to the `wp core language update` command after the `remove_action` function:
```php
\WP_CLI::run_command( array( 'core', 'language', 'update' ), array( 'dry-run' => false ) );
```
You still need to run the command manually after the update, because after the plugin/theme update it will automatically check for translations updates to the older version of the plugin/theme instead of the new. Its the same behaviour as the default method, so its seems like a WordPress bug.